### PR TITLE
Added dependency on common-ngx from the deleteProjectDependencies tas…

### PR DESCRIPTION
**Pull request by bgsavage**
…k.  This ensures that when gradle buildFe is called, that common-ngx is also built.

### CODE REVIEW

File: webapp/build.gradle [hunk 1]
```diff
@@ -63,6 +63,7 @@ test.finalizedBy(jacocoTestReport)
 
 task deleteProjectDependencies(type: Delete) {
     delete "src/main/webapp/node_modules/@sbac/rdw-reporting-common-ngx"
+    dependsOn project(':rdw-reporting-common-ngx').tasks.buildFe
 }
```
Comment (bgsavage) [inline webapp/build.gradle]: I created this as a separate PR as I didn't want this to be buried in all my other feature changes in the previous pr.

gradle buildFe will now trigger a common-ngx buildFe.

Comment (chdietz) [reply webapp/build.gradle]: every time or only if it has changed?

Comment (esotrope) [reply webapp/build.gradle]: the addition of `inputs/outputs` to the common-ngx `build.gradle` config supposedly allow caching/change watching to work so gradle can reuse builds if the contents of those haven't changed

Comment (bgsavage) [reply webapp/build.gradle]: This is what I've observed as well.

Comment (bgsavage) [reply webapp/build.gradle]: What's crazy is this same exact line of code in webapp-admin throws this error:  
```
 Could not get unknown property 'buildFe' for task set of type org.gradle.api.internal.tasks.DefaultTaskContainer.
```
Any thoughts?

File: admin-webapp/build.gradle [hunk 2]
```diff
@@ -69,6 +69,11 @@ bootRun {
 // generate coverage report automatically (./build/reports/jacoco/test/html/index.html)
 test.finalizedBy(jacocoTestReport)
 
+task deleteProjectDependencies(type: Delete) {
+    delete "src/main/webapp/node_modules/@sbac/rdw-reporting-common-ngx"
+    dependsOn project(':rdw-reporting-common-ngx').tasks.build
```
Comment (bgsavage) [inline admin-webapp/build.gradle]: `build` seems to work in admin, though I still don't understand why `buildFe` doesn't work.

Any gradle experts can enlighten me here?  @wtritch @chdietz @malaffoon  ?

Also, should I update the webapp gradle to use `build` as well for consistency?  `buildFe` in theory should be slightly more efficient and as it won't try to compile java code that's not present in common-ngx.

**Pull request merged**